### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'toLowerCase')

### ISSUE THREAD

**Issue by WangJincheng4869** [siaikin/vue-to-print#15]
**Describe the bug**
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    at _ (vue-to-print.js?t=1731897420165&v=b21db612:250:42)
    at vue-to-print.js?t=1731897420165&v=b21db612:359:23
    at Generator.next (<anonymous>)
    at vue-to-print.js?t=1731897420165&v=b21db612:38:25
    at new Promise (<anonymous>)
    at A (vue-to-print.js?t=1731897420165&v=b21db612:24:22)
    at Ce (vue-to-print.js?t=1731897420165&v=b21db612:356:10)
    at vue-to-print.js?t=1731897420165&v=b21db612:463:21
    at Generator.next (<anonymous>)
    at vue-to-print.js?t=1731897420165&v=b21db612:38:25
```

![图片](https://github.com/user-attachments/assets/b7a7e625-7de1-4aac-b393-97d691d3bb7b)

猜测应该是这个函数。

![图片](https://github.com/user-attachments/assets/18cf3df1-4c12-44d8-9027-9c37eadaa0ca)

**Comment by siaikin** [comment]
Hello, thank you so much for taking the time to submit an issue.
I'll check under what circumstances 'nodeName' is undefined.

Can you provide an online example to reproduce the issue? This can help me solve this problem better. There's an example of stackblitz on the README that you can modify on top of that.

**Comment by WangJincheng4869** [comment]
@siaikin 我尝试将自定义的组件进行打印

![图片](https://github.com/user-attachments/assets/454942ae-39db-4f78-80d1-5a47b78ffda1)

`ErpMaterialOutboundDetails` 组件内的内容
![图片](https://github.com/user-attachments/assets/d00abb8d-1cd7-4076-a222-395f360053d0)

我使用了 `ElDrawer` 打开详情，它是 [element-plus](https://element-plus.org/zh-CN/component/drawer.html) 组件。

**Comment by siaikin** [comment]
Hi, `content` should be a value of type HTMLElement. This is because vue3 can't get the real DOM elements from a component's references.

like this:
```html
<div ref="componentRef">
    < ErpMaterialOutboundDetails :data="data" />
  </div>
```

I'll clarify this in the documentation, and thank you very much for your feedback.

**Comment by siaikin** [comment]
I fixed this and released the '1.3.0' version. Please try again.
